src/inet/ZephyrSocket.h

```diff
@@ -31,6 +31,7 @@
 #include <zephyr/net/socket.h>
 #endif
 
+#ifndef CONFIG_NET_CONTEXT_RECV_PKTINFO
```

Comment:
You can just disable `CHIP_SYSTEM_CONFIG_USE_ZEPHYR_SOCKET_EXTENSIONS` for your platform if you don't need this extension. We will probably get rid off this file when nRF and Telink update Zephyr.

Reply:
I didn't catch this flag, thanks for pointing it out !
I'll try that instead, and decline if that works.

Reply:
This solution works for us, I'm closing the PR.